nvflare/app_opt/xgboost/histogram_based_v2/runners/xgb_client_runner.py

```diff
@@ -95,6 +99,17 @@ def _xgb_train(self, num_rounds, xgb_params: dict, xgb_options: dict, train_data
         early_stopping_rounds = xgb_options.get("early_stopping_rounds", 0)
         verbose_eval = xgb_options.get("verbose_eval", False)
 
+        # Check for pre-trained model
+        job_id = self.fl_ctx.get_prop(FLContextKey.CURRENT_JOB_ID)
+        workspace = self.fl_ctx.get_prop(FLContextKey.WORKSPACE_OBJECT)
+        custom_dir = workspace.get_app_custom_dir(job_id)
+        model_file = os.path.join(custom_dir, MODEL_FILE_NAME)
+        if os.path.isfile(model_file):
+            self.logger.info(f"Pre-trained model is used: {model_file}")
+            xgb_model = model_file
+        else:
+            xgb_model = None
```

Comment:
even if a model file exist does that mean users want to use it?
if so, we need to update the documentation specifying "if they have a model_file inside the custom folder it will be used."

Reply:
Yeah. We decided to use it as pre-trained model if one exists in custom folder.